tests/jax/targets.jsonnet

```diff
@@ -16,9 +16,11 @@ local compilationCacheTest = import 'compilation-cache.libsonnet';
 local latest = import 'latest/targets.jsonnet';
 local nightly = import 'nightly/targets.jsonnet';
 local podTest = import 'pod-test.libsonnet';
+local tpu_embedding = import 'tpu-embedding.libsonnet';
```

Comment:
It seems we have a historical issue about local variable naming. In this case, I'd suggest we just follow up the pattern, and name it as something like `local tpuEmbedding = import 'tpu-embedding.libsonnet'`. 

Also, do we have nightly tests for JAX? If not, please help us remove `local nightly = import 'nightly/targets.jsonnet'` as an extra effort.